R/prophet.R

```diff
@@ -204,12 +204,18 @@ do_prophet_ <- function(df, time_col, value_col = NULL, periods, time_unit = "da
       ret <- ret %>% dplyr::select(-cap_scaled)
     }
     # TODO: Maybe we should take average when MCMC is used and there are multiple delta values for each channge point.
-    changepoints_df <- data.frame(ds = m$changepoints, slope_change = m$params$delta[1,])
-    # m$changepoints is POSIXct. Cast it to Date when necessary so that left_join works.
-    if (lubridate::is.Date(aggregated_data$ds)) {
-      changepoints_df$ds <- as.Date(changepoints_df$ds)
+    if (!is.numeric(m$changepoints)) { # m$changepoints seems to become numeric single 0 when empty.
+      changepoints_df <- data.frame(ds = m$changepoints, slope_change = m$params$delta[1,])
+      # m$changepoints is POSIXct. Cast it to Date when necessary so that left_join works.
+      if (lubridate::is.Date(aggregated_data$ds)) {
```

Comment:
Shouldn't be `is.POSIXct`?

Reply:
It is is.numeric. I tried to explain why the comment.

Reply:
? I'm talking about this part. 

![image](https://user-images.githubusercontent.com/68080/40933124-6a01bde0-67e5-11e8-89b7-fcd60623c0c9.png)

Reply:
I see. Updated the comment so that this is less confusing.